core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1114,16 +1114,6 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
-  private[spark] val STORAGE_LOCAL_DISK_BY_EXECUTORS_CACHE_SIZE =
-    ConfigBuilder("spark.storage.localDiskByExecutors.cacheSize")
```

Comment:
good catch! It's weird to see a storage config in the middle of a bunch shuffle configs.